### PR TITLE
fix(react-scheduler): add capability to pass className and style into Scheduler layout component

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**

### CODE REVIEW

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.jsx [hunk 1]
```diff
@@ -70,4 +73,9 @@ HorizontalViewLayoutBase.propTypes = {
   height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
 };
 
+
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.jsx]: Avoid adding excess new lines

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.jsx [hunk 2]
```diff
@@ -68,6 +73,13 @@ HorizontalViewLayoutBase.propTypes = {
   layoutRef: PropTypes.object.isRequired,
   layoutHeaderRef: PropTypes.object.isRequired,
   height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+  className: PropTypes.string,
+  style: PropTypes.object,
+};
+
+HorizontalViewLayoutBase.defaultProps = {
+  className: undefined,
+  style: undefined,
```
Comment (ushkal) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.jsx]: We use `null` as a default value for object props e.g. `style`

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    classes = getClasses(<HorizontalViewLayout {...defaultProps} />);
+    shallow = createShallow({ dive: true });
+  });
+
+  it('should pass className to the root element', () => {
+    const tree = shallow((
+      <HorizontalViewLayout {...defaultProps} className="custom-class" />
+    ));
+
+    expect(tree.find('.custom-class'))
```
Comment (ushkal) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx]: We should check root component in this tests: `tree.hasClass('custom-class')`

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    expect(tree.find(`.${classes.container}`))
+      .toBeTruthy();
+  });
+
+  it('should pass rest props to the root element', () => {
+    const tree = shallow((
+      <HorizontalViewLayout {...defaultProps} data={{ a: 1 }} />
+    ));
+
+    expect(tree.find(`.${classes.container}`).props().data)
```
Comment (ushkal) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx]: restProps should be applied to the root element so we should check `tree`: `tree.prop('data') ...`

Comment (ushkal) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx]: restProps should be applied to the root element so we should check `tree`: `tree.prop('data') ...`

Comment (ushkal) [reply packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx]: Same applies to other tests

**Pull request merged**